app.js

```diff
@@ -222,27 +224,26 @@ aws.initConfig(awsProps)
             clientOptionsHandler);
 
         function oldGoogleUrlHandler(req, res, next) {
-            const resolver = new google.ShortLinkResolver(aws.getConfig('googleApiKey'));
+            //const resolver = new google.ShortLinkResolver(aws.getConfig('googleApiKey'));
```

Comment:
we can acutally move this code into the `google.ShortLinkResolver` itself and then the `app.js` code does'nt need to change at all :)

Reply:
I think :)